.github/workflows/rust.yml

```diff
@@ -350,3 +350,45 @@ jobs:
           pip install maturin==0.8.2 toml==0.10.1 pyarrow==1.0.0
           maturin develop
           python -m unittest discover tests
+
+  # test the arrow crate builds against wasm32 in stable rust
+  wasm32-build:
+    name: AMD64 Debian 10 Rust ${{ matrix.rust }} test arrow wasm32
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64]
+        rust: [nightly-2020-11-24]
+    container:
+      image: ${{ matrix.arch }}/rust
+      env:
+        ARROW_TEST_DATA: /__w/arrow/arrow/testing/data
+        PARQUET_TEST_DATA: /__w/arrow/arrow/cpp/submodules/parquet-testing/data
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Cache Cargo
+        uses: actions/cache@v2
+        with:
+          path: /github/home/.cargo
+          # this key equals the ones on `linux-build-lib` for re-use
+          key: cargo-cache-
+      - name: Cache Rust dependencies
+        uses: actions/cache@v2
+        with:
+          path: /github/home/target
+          key: ${{ runner.os }}-${{ matrix.arch }}-target-wasm32-cache-${{ matrix.rust }}
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          default: true
+          override: true
+          components: rustfmt
+          target: wasm32-unknown-unknown
+      - name: Build arrow crate
```

Comment:
I wonder if we could also run some tests? Or maybe that is a follow on step?

Reply:
Follow-up, as some tests currently fail